Source/debug.cpp

```diff
@@ -505,7 +532,8 @@ std::string DebugCmdQuest(const string_view parameter)
 
 std::string DebugCmdLevelUp(const string_view parameter)
 {
-	int levels = std::max(1, atoi(parameter.data()));
+	const ParseIntResult<int> parsedArg = ParseInt<int>(parameter, /*min=*/1);
+	const int levels = parsedArg.ok() ? parsedArg.value : 1;

```

Comment:
`ParsedIntResult<T>::value_or(T value_if_error)` would be a great helper for cases like this, matches https://en.cppreference.com/w/cpp/utility/expected/value_or if c++23 support becomes a possibility :D

Reply:
Done! We might as well add `tl::expected` once we've switched to C++17